.github/workflows/go-test.yml

```diff
@@ -8,26 +8,24 @@ on:
 
 jobs:
   test:
-    name: Go 1.19
+    name: Go ${{ matrix.go-version }}
+
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
I'm not sure whether supporting different Go versions for our tool is something we need. It's a _tool_, ideally used only as a binary. Compiling it from source is one of the ways to install it, but I wouldn't call it something we officially support

Reply:
> I'm not sure whether supporting different Go versions for our tool is something we need.

Strong agree.

> Compiling it from source is one of the ways to install it, but I wouldn't call it something we officially support

Strong agree as well.

Let's use the latest stable Go release :)